docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -1347,6 +1347,12 @@ function layout_builder_custom_preprocess_views_view_unformatted(&$variables) {
         'card_date' => '',
         'headline_level' => $view->display_handler->getOption('heading_size'),
       ];
+
+      // Only if the events card_list view display is used (sub events).
+      if ($view->id() === 'events' && $machine_name === 'card_list') {
```

Comment:
Can you set this on the block config for the event page? Or is this something else you are trying to do?

Reply:
Maybe the enclosed but the config options weren't there. Needed this handle this content type specific field mapping though.

Reply:
>but the config options weren't there

Correction: The block config is not currently moving card--enclosed down to individual items
<img width="894" alt="Screen Shot 2022-10-31 at 1 06 18 PM" src="https://user-images.githubusercontent.com/4663676/199078466-734686f1-c0be-4f6c-b263-41a006dad66b.png">

Reply:
@bspeare can this be resolved?